server/concepts/Tournament.ts

```diff
@@ -202,14 +203,15 @@ export default class Tournament implements TournamentWithoutMS {
     await TournamentRepository.delete(tournamentId);
   }
 
-  private static generateRoundRobinMatches(teams: Required<TeamWithoutMS>[]): Match[] {
+  private static generateRoundRobinMatches(teams: Required<TeamWithoutMS>[]): MatchRoundRobinCreate[] {
     const { data } = tournamentGenerator([...teams], { type: 'single-round' });
 
     // generowanie dat
-    return data.map((match) => Match.getNewInstance({
+    const matches = data.map((match) => Match.getNewInstance({
       teamA: match.homeTeam,
       teamB: match.awayTeam,
-    }));
+    })) as MatchRoundRobinCreate[];
+    return matches;
```

Comment:
I think we can back to old code with return and leave "as ..."

Reply:
changed